### ISSUE TITLE
NTR, pulmo: Elevated level of lymphocytes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5561]
**Preferred term label:**
Elevated level of lymphocytes

**Synonyms**
Elevated level of lymphocytes in broncho-alveolar lavage fluid

**Definition (free text, please give PubMed ID)**
Usually, Lymphoycytes make up less than 15 % of all cells found in the broncho-alveloar lavage fluid. This elevated cell proportion can be induced by virus or drugs, or is associated to specific diseases. 
(Costabel ISBN0412792702; PMID: 3396403; PMID: 10678650)

**Parent term (use hpo.jax.org/app)**
Abnormal cellular composition of bronchoalveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:797, ORPHA:31740, ORPHA:411703, ORPHA:1302, ORPHA:747, ORPHA:264675, ORPHA:182098, OMIM:601495, OMIM:266600, OMIM: 611081, OMIM: 612278, ORPHA:69126, ORPHA:186

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Adding as Elevated bronchoalveolar lavage fluid lymphocyte proportion
